terraform/modules/rds/parameters.tf

```diff
@@ -1,6 +1,60 @@
 resource "aws_ssm_parameter" "database_partial_connection_string" {
   name   = "DATABASE_PARTIAL_CONNECTION_STRING"
-  key_id = aws_kms_key.this.arn
+  key_id = aws_kms_key.ssm.arn
   type   = "SecureString"
   value  = "postgresql://${aws_db_instance.this.username}:${aws_db_instance.this.password}@${aws_db_instance.this.endpoint}"
 }
+
+resource "aws_kms_key" "ssm" {
+  description         = "KMS key used to decrypt SSM parameters."
+  enable_key_rotation = true
+}
+
+moved {

```

Comment:
These moved blocks can be removed once this has been applied to all environments.